### PR TITLE
[THORN-2116] Adding the additional resources info

### PR DESCRIPTION
**Pull request by sberyozkin**

### CODE REVIEW

**Comment by sberyozkin** [comment]
Please see https://wildfly-swarm.gitbooks.io/wildfly-swarm-users-guide/reading.html
Not sure all those links have to be included, two per each sub section should be enough, but can easily add more links

**Comment by Ladicek** [comment]
I'd totally recommend including a link to the Sam Newman's book, it's really good.

File: docs/modules/ref_additional-resources.adoc [hunk 1]
```diff
@@ -0,0 +1,20 @@
+
+[id='additional-resources_{context}']
+= Additional resources
+
+Please follow the links below to find out more about Thorntail and the Microservices Architecture
```
Comment (tradej) [inline docs/modules/ref_additional-resources.adoc]:
```suggestion
Follow the links below for information about Thorntail and the Microservices Architecture:
```

File: docs/modules/ref_additional-resources.adoc [hunk 2]
```diff
@@ -0,0 +1,20 @@
...
+
+[discrete]
+== Thorntail
+
+* thorntail.io[Homepage]
```
Comment (tradej) [inline docs/modules/ref_additional-resources.adoc]:
```suggestion
* Homepage: https://thorntail.io[thorntail.io]
```

File: docs/modules/ref_additional-resources.adoc [hunk 3]
```diff
@@ -0,0 +1,20 @@
...
+* https://thorntail.io/archive/[Blog]
```
Comment (tradej) [inline docs/modules/ref_additional-resources.adoc]:
```suggestion
* Blog: https://thorntail.io/archive/[thorntail.io/archive]
```

File: docs/modules/ref_additional-resources.adoc [hunk 4]
```diff
@@ -0,0 +1,20 @@
...
+
+[discrete]
+== Microservices
+
+* http://shop.oreilly.com/product/0636920033158.do[Sam Newman, "Building Microservices"]
```
Comment (tradej) [inline docs/modules/ref_additional-resources.adoc]:
```suggestion
* Book: http://shop.oreilly.com/product/0636920033158.do[Sam Newman, "Building Microservices"]
```

File: docs/modules/ref_additional-resources.adoc [hunk 5]
```diff
@@ -0,0 +1,20 @@
...
+* https://www.manning.com/books/enterprise-java-microservices[Ken Finnigan, "Enterprise Java Microservices"]
```
Comment (tradej) [inline docs/modules/ref_additional-resources.adoc]:
```suggestion
* Book: https://www.manning.com/books/enterprise-java-microservices[Ken Finnigan, "Enterprise Java Microservices"]
```

File: docs/modules/ref_additional-resources.adoc [hunk 6]
```diff
@@ -0,0 +1,20 @@
...
+* https://www.thoughtworks.com/de/insights/blog/microservices-nutshell[Martin Fowler, "Microservices in a Nutshell"]
```
Comment (tradej) [inline docs/modules/ref_additional-resources.adoc]:
```suggestion
* Blog: https://www.thoughtworks.com/de/insights/blog/microservices-nutshell[Martin Fowler, "Microservices in a Nutshell"]
```

**Review by tradej — CHANGES_REQUESTED** [review]
Small changes requested.

**Comment by sberyozkin** [comment]
Sheldon, thanks for the suggestions, applied

**Comment by sberyozkin** [comment]
Looks ready for the merge too, can someone please approve, @Ladicek, @michalszynkiewicz

**Review by Ladicek — APPROVED** [review]

**Pull request merged**